lib/workflows/utils.js

```diff
@@ -68,6 +69,37 @@ const check = (config) => {
   return test[0];
 };
 
+/**
+ * Get a single workflow from all workflows available
+ *
+ * @param {object} type - a content type object
+ * @param {object} workflows - an object of all workflows
+ *
+ * @returns {promise} reject with error message; resolve with single workflow object
+ */
+const workflow = (type, workflows) => {
+  return new Promise((res, rej) => {
+    let wf;
+    if (type.workflow) {
+      wf = workflows.find(flow => {
+        return flow.id === type.workflow;
+      });
+    }
+    else {
+      wf = workflows.find(flow => {
+        return flow.id === globalConfig.workflows.default;
+      });
+    }
+
+    if (wf === undefined || typeof wf !== 'object') {
```

Comment:
This just needs to be `typeof wf !== 'object`. If `wf` is undefined, it'll be `typeof undefined`